packages/api/src/controllers/stream.ts

```diff
@@ -770,6 +781,7 @@ app.post(
       profiles: childStream.profiles,
       record,
       recordObjectStoreId,
+      recordingSessionId,
       recordingStatus: record ? "waiting" : undefined,
     };
     await db.session.create(session);
```

Comment:
I think you're missing some logic to check if the session doesn't already exist before creating a new one.

Reply:
My understanding is that we still keep the relation of 1-1 between the child stream and session. Then, I think we'll always create a session here. Isn't that the case?

Reply:
I had the exact other understanding as explained in the other comment and on that Discord thread. We should have 1-N relationship between sessions and child streams of the same session.